python/ray/serve/master.py

```diff
@@ -138,6 +138,8 @@ def _list_replicas(self, backend_tag):
 
         # Wait for the worker to start up.
         await worker_handle.ready.remote()
+        await self.get_router()[0].add_new_worker.remote(
```

Comment:
```
_, _ = self.get_router
await _.add_new_worker...
```

Reply:
In this case it's just `[router_handle]` so I wrote it this way. We only need this because ray blocks you from returning actor handles directly...